lib/rex/post/postgresql/ui/console/command_dispatcher/core.rb

```diff
@@ -0,0 +1,50 @@
+# -*- coding: binary -*-
+
+module Rex
+  module Post
+    module PostgreSQL
+      module Ui
+
+        ###
+        #
+        # Core SMB client commands
```

Comment:
```suggestion
        # Core Postgresql client commands
```

There is probably more I haven't checked.